app/components/Navbar.tsx

```diff
@@ -0,0 +1,108 @@
+import { Disclosure, DisclosureButton, DisclosurePanel } from '@headlessui/react'
+import { Bars3Icon, UserCircleIcon, XMarkIcon, ClipboardDocumentListIcon, Bars3CenterLeftIcon } from '@heroicons/react/24/outline'
+import Link from 'next/link'
+import { useState } from 'react';
+import LoginRegisterCard from './LoginRegisterCard';
+
+const navigation = [
+    { name: 'Manage my bookings', href: '/manage-my-booking-info', current: false, icon: ClipboardDocumentListIcon },
+    { name: 'Login | Register', href: '/', current: false, icon: UserCircleIcon },
+]
+
+function classNames(...classes) {
```

Comment:
This can be global function in utils folder for example.
Check this article please https://medium.com/@nomanfareed681/simplify-your-tailwind-css-class-management-with-merge-and-clsx-42f1e2458fd8